packages/runtime/container-runtime/src/opLifecycle/opDecompressor.ts

```diff
@@ -126,8 +129,12 @@ export class OpDecompressor {
 	 * Unroll the next message from the decompressed content provided to {@link decompressAndStore}
 	 * @returns the unrolled `ISequencedDocumentMessage`
 	 */
-	public unroll(message: ISequencedDocumentMessage): ISequencedDocumentMessage {
-		assert(this.currentlyUnrolling, 0x942 /* not currently unrolling */);
+	public unroll(message: ISequencedDocumentMessage): {
+		message: ISequencedDocumentMessage;
+		batchStartCsn: number;
+	} {
```

Comment:
Nit: Can we type this to be something?

Reply:
You mean name it? I guess, would that help readability?

Reply:
Yeah, it would help a little :)

Reply:
Reverted this change actually